plugins/module_utils/common.py

```diff
@@ -663,6 +677,14 @@ def build_error_msg(kind, name, msg):
                 return result
 
         else:
+            if label_selectors:
+                filter_selector = LabelSelectorFilter(label_selectors)
+                if not filter_selector.isMatching(definition):
+                    result['changed'] = False
+                    del result['result']
+                    result['warning'] = "resource 'kind={kind},name={name},namespace={namespace}' filtered by label_selectors.".format(
```

Comment:
This isn't really a warning because it's exactly what the user requested. I'd suggest we instead use `msg` here.